docs/source/about/cite.rst

```diff
@@ -0,0 +1,35 @@
+How to cite us
+==============
+If you find CellRank useful for your research, please consider citing :cite:`lange:22` as:
+
+.. code-block:: bibtex
+
+    @article{lange:22,
+        title = {CellRank for directed single-cell fate mapping},
+        author = {Lange, Marius and Bergen, Volker and Klein, Michal and Setty, Manu and
+                  Reuter, Bernhard and Bakhti, Mostafa and Lickert, Heiko and
+                  Ansari, Meshal and Schniering, Janine and Schiller, Herbert B. and
+                  Pe'er, Dana and Theis, Fabian J.},
+        journal = {Nat. Methods},
+        year = {2022},
+        doi = {10.1038/s41592-021-01346-6},
+        publisher = {Nature Publishing Group}
+    }
+
+In addition, if you use :class:`cellrank.estimators.GPCCA` to compute initial, terminal or intermediate states, you are
+using the `pyGPCCA package <https://pygpcca.readthedocs.io/>`_ :cite:`reuter:22` under the hood, which implements the
```

Comment:
@Marius1311, you are still pointing to RTD here. I guess pointing to GitHub (https://github.com/msmdev/pyGPCCA) here, might be more appropriate?

Reply:
Ok, I'll adapt this.